src/components/charts/ReferencesTable.tsx

```diff
@@ -0,0 +1,108 @@
+import _ from 'lodash';
+import React, { useContext, useState, useEffect } from "react";
+import { Table, Pagination } from "semantic-ui-react";
+import { AppContext } from "../../context";
+import './Charts.css';
+
+export default function ReferencesTable() {
+  const [column, setColumn] = useState('');
+  const [state, dispatch] = useContext(AppContext);
+  const [activePage, setActivePage] = useState(1);
+  const [boundaryRange, setBoundaryRange] = useState(1);
+  const [siblingRange, setSiblingRange] = useState(1);
+  const [totalPages, setTotalPages] = useState(10);
+
+    const handlePaginationChange = (e : any, event: any) => { 

```

Comment:
maybe document why there need to be two events in the function handle

Reply:
I don't know why, but for some reason this is how it comes